wapitiCore/attack/mod_cookieflags.py

```diff
@@ -0,0 +1,43 @@
+import requests
+from wapitiCore.attack.attack import Attack
+from wapitiCore.net.web import Request
+from wapitiCore.language.vulnerability import Additional
+
+class mod_cookieflags(Attack):
+
+    name = "cookieflags"
+
+    def check_secure_flag(self, cookie: object):
+
+        return cookie.secure
+
+    def check_httponly_flag(self, cookie: object):
+
+        return 'HttpOnly' in cookie._rest
+
+    def attack(self):
+        url = self.persister.get_root_url()
+        session = requests.session()
+        _req = session.get(url)
+        request = Request(url)
+        for cook in session.cookies:
+            self.log_blue("Check HttpOnly flag :")
+            if not self.check_httponly_flag(cook):
+                self.log_red("HttpOnly flag is not set")
```

Comment:
Use _("...") format to support translations

Reply:
The log function it's just to display the status on the Terminal, it's not a variable.
And when i try to use the _("...") format i get this
![format](https://user-images.githubusercontent.com/58309137/88303336-3708a500-cd07-11ea-9d65-2b4998732781.JPG)

Reply:
You can create a _("...") string in vulnerability.py that you can format using self.log_{color}(string, param1, prama2, ...)